cmd/plural/config.go

```diff
@@ -64,7 +64,7 @@ func handleAmend(c *cli.Context) error {
 	return config.Amend(c.Args().Get(0), c.Args().Get(1))
 }
 
-func handleRead(c *cli.Context) error {
+func handleRead(*cli.Context) error {
```

Comment:
this change looks a bit strange, especially given that the `handleConfigImport` function below looks essentially the same and doesn't have this change.

Reply:
yes you are right, linter doesn't complain

Reply:
I guess both of them should have the variable name removed since they do not use context. It's strange that the `handleConfigImport` was not picked up by the linter.

Reply:
generally, the parameter is required to fulfill interface requirements. Linter suggests using just the type.